src/Plugin/Plugins/TempHumidPlugin.h

```diff
@@ -0,0 +1,242 @@
+/* MIT License
+ *
+ * Copyright (c) 2019 - 2021 Andreas Merkle <web@blue-andi.de>
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+/*******************************************************************************
+    DESCRIPTION
+*******************************************************************************/
+/**
+ * @brief  Reads temperature and humidity from a sensor and displays it
+ * @author Flavio Curti <fcu-github@no-way.org>
+ * @addtogroup plugin
+ *
+ * @{
+ */
+
+#ifndef __TEMPHUMIDPLUGIN_H__
+#define __TEMPHUMIDPLUGIN_H__
+
+/******************************************************************************
+ * Compile Switches
+ *****************************************************************************/
+#define DHTTYPE DHTesp::DHT11   // DHT 11 sensor
+#define DHTTYPE DHTesp::DHT11   // DHT 11 sensor
```

Comment:
Why is it defined twice?
Please use doxygen comments.

Reply:
copy paste error.